client/src/theme/styles/components/frontend/project/_resource-thumbnail-list.scss

```diff
@@ -1,30 +1,57 @@
 .resource-thumbnail-list {
   ul {
     @include listUnstyled;
+
+    // flex fallback
     display: flex;
     flex-wrap: wrap;
+    // push it 20px to the left to negate the left padding on the resources
     margin-left: -20px;
-  }
 
-  li {
-    display: flex;
-    width: 50%;
-    padding-left: 20px;
-    margin-bottom: 20px;
-
-    &:nth-last-of-type(-n + 2) {
-      display: none;
+    li {
+      // 2 columns
+      width: 50%;
+      // 20px padding on the bottom and left
+      padding: 0 0 20px 20px;
+      @include respond($break65) {
+        // 4 columns
+        width: 25%;
+      }
+      @include respond($break90) {
+        // 5 columns
+        width: 20%;
+      }
     }
 
-    @include respond($break65) {
-      width: 25%;
-    }
+    // grid style if the browser supports it
+    @supports ( grid-area: auto ) {
+      display: grid;
+      // use grid gap instead
+      grid-gap: 20px;
+      grid-template-rows: repeat(2, auto);
+      grid-template-columns: repeat(2, 1fr);
+      // remove the margin from the fallback
+      margin-left: 0;
 
-    @include respond($break90) {
-      width: 20%;
+      li {
```

Comment:
Make sure all the `li` elements have matching heights.

![Screen Shot 2019-05-01 at 9 22 38 AM](https://user-images.githubusercontent.com/12454203/57028781-d61e1a00-6bf4-11e9-8063-d040a7b7cb09.png)